src/Evolution/Systems/Cce/Initialize/RegisterInitializeJWithCharm.hpp

```diff
@@ -16,10 +16,11 @@ struct LinearizedBondiSachs;
 
 /// A function for registering all of the InitializeJ derived classes with
 /// charm, including the ones not intended to be directly option-creatable
+template <bool uses_inertial_coordinates>
```

Comment:
Should this template parameter be renamed `uses_partially_flat_cartesian_coordinates`?